### ISSUE TITLE
Wow that is a big spam of 403 issues

### ISSUE THREAD

**Issue by pato-pan** [yuliskov/SmartTube#3665]
someone has to step up and keep this organized, keep everything in one place. This is crazy

**Comment by Farmerfds** [comment]
You are right!  Good idea to start another one...lol.  Sorry, could not help it.   
Would be nice to have one 403 post with everything.

**Comment by faroukbmiled** [comment]
> 

Why opening another issue for this, come on now

**Comment by zenithyr** [comment]
you can help by closing this one first :)

**Comment by pato-pan** [comment]
Thank you for appreciating my genius

**Comment by pato-pan** [comment]
if this is resolved someone else should close it. It shouldn't be closed until someone steps up to manage these issues, because the dev can't, this is too much and they are busy.

**Comment by pato-pan** [comment]
like the fact that you **need** **_me_** to close it clearly shows there's no one doing this right now.

You know what? Just give me whatever it is that allows me to close and manage these issues and I will do what I can. I don't have a lot of free time, but I probably have more time than others if no one is going to volunteer to do this.

**Comment by Alpen1** [comment]
He isnt wrong. There spam and countless duplicate messages. Someone needs to do a cleanup

**Comment by HelpfulCarrot** [comment]
In addition to keeping things organized, consolidating all these duplicate issue threads would allow people to more easily find [this comment with a helpful workaround](https://github.com/yuliskov/SmartTube/issues/3606#issuecomment-2214985225) so that people can continue using this fantastic app by fantastic devs who cannot reasonably be expected to spend more time than they already do on this project! So I think it's great that @pato-pan wants to step up, as long as assigning them the collaborator role doesn't pose any security risks.

**Comment by pato-pan** [comment]
[In regards to security risks](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/permission-levels-for-a-personal-account-repository#collaborator-access-for-a-repository-owned-by-a-personal-account)

**Comment by HelpfulCarrot** [comment]
True, and there's a 99.99% chance you're a well-meaning volunteer, but because of things like the [XZ Utils backdoor (see this great writeup in Wired)](https://www.wired.com/story/jia-tan-xz-backdoor/), I felt I had to mention that security is the only downside I can think of. But I'm not connected to this repo in any way, so my opinion doesn't really matter. However... (see next comment)

**Comment by HelpfulCarrot** [comment]
I will tag @yuliskov so that, when Yuriy has a chance to see this, he will be made aware that you have offered to help him manage the issue tracker.

Summary for Yuriy: pato-pan has offered to step in to manage the Issues page, consolidating all the '403 error' issue threads, which would also allow users to more easily find the workarounds. There may be some security risks in assigning someone the collaborator role, but other than that there doesn't seem to be a downside, and you clearly have your hands full maintaining this wonderful project. Also, if you see this, thanks so much for all the work you (and other devs?) do! It's a truly wonderful project.
